### ISSUE TITLE
Allow tabs on tab to be loaded slowly

### ISSUE THREAD

**Issue by philikon** [philikon/BarTab#7]
Allow tabs that have been put on tab to be loaded slowly, one by one. That way you still get a speedy Firefox restart _and_ all your tabs will be available soon after.

**Comment by qwerty017** [comment]
Take a look at Load Tabs Progressively.This does exactly what you are talking about.

**Comment by philikon** [comment]
Yep, I know of this extension. I'm still not quite sure this should be part of BarTab itself.

**Comment by qwerty017** [comment]
Yeah. I was just telling philikon about it in case he wanted the option now.

**Comment by qwerty017** [comment]
Whoops. lol. didn't notice you were philikon. lol
